### PR TITLE
Template task order test

### PR DESCRIPTION
**Pull request by thamaranth**
Fixes # .

## Overview

(quick description of solution to original issue)

## Data Model / DB Schema Changes

(if there were data changes, explain them)
(also note whether it is necessary to run any database migrations)
(N/A if it doesnt apply)

## Environment / Configuration Changes

(any updated third-party libraries / packages?)
(any updates to the configuration files?)
(N/A if it doesnt apply)

## Notes

(heads up on anything not super intuitive or obvious)

### CODE REVIEW

File: src/browser/components/atoms/List.js [hunk 1]
```diff
@@ -14,19 +14,9 @@ export default class List extends Component{
...
-            <td>
-              <button type="button" className="btn btn-default"
-                aria-label="Left Align" onClick={this.props.deleteTaskCallback.bind(this, task)}>
-                Delete
-              </button>
-            </td>
-          </tr>
-        )}
+          { (this.props.currentTemplateTasks || []).map((task) =>
+              <ListItem task={task} {...this.props} />
```
Comment (punitrathore) [inline src/browser/components/atoms/List.js:18]: Why is this code in this PR? Was this missed in the previous PR ?

File: src/database/queries/template_task.js [hunk 2]
```diff
@@ -23,4 +23,16 @@ const update = ( id, attributes ) =>
 const expunge = ( column, data ) =>
   _.deleteRecord( 'template_task', column, data )
 
-export { add, deleteAll, getAll, getBy, update, expunge, deleteRecord }
+const convert = github_handle => {
+  return user.create({
+    full_name: "Ugly Face",
+    github_handle: "Trump_Butt",
```
Comment (punitrathore) [inline src/database/queries/template_task.js]: Appreciate the sentiment, but can we please have civilized names please :)

File: src/browser/components/atoms/List.js (hunk 1 above)
Comment (thamaranth) [reply src/browser/components/atoms/List.js:18]: Yes

File: src/database/queries/template_task.js (hunk 2 above)
Comment (thamaranth) [reply src/database/queries/template_task.js]: Totally not my naming convention, it can be changed

**Comment by thamaranth** [comment]
changes made

**Comment by punitrathore** [comment]
@thamaranth looks good, merging this.

**Pull request merged**